### PR TITLE
anchor link button and form field tweaks

### PR DESCRIPTION
**Pull request by martinbagshaw**
- Noticed I hadn't set colour for :visited on anchor links that look like buttons (as on the home page). This is now fixed.
- Also added better spacing for form fields in the manual data entry section

### CODE REVIEW

**Comment by martinbagshaw** [comment]
@wright1 I have added an upload pdf button with the + icon in to this pr.
Also added a way to pass a prop in to get a purple button for icons, like so:

`<IconButton
   isSolid
   aria-label='Add something here if there is no text'>  
<Plus fill={colors.white} width={48} />
</IconButton>
`

File: src/components/pages/home/index.js
Comment (MohammedYehia) [inline src/components/pages/home/index.js]: this is the same sentence 
do we need both of them?

**Review by MohammedYehia — CHANGES_REQUESTED** [review]

File: src/components/pages/home/index.js
Comment (martinbagshaw) [reply src/components/pages/home/index.js]: I added both buttons to serve as an example of how to pass in `isSolid` to give an icon button with a purple background. Will remove. Same reasoning behind leaving the route in with the icons on the card.

@susanX @wright1 @othman-shamla would you like me to create an issue for how to pass in props to style buttons and icons, or can you figure this out ok?

Comment (martinbagshaw) [reply src/components/pages/home/index.js]: @MohammedYehia changes made

**Comment by wright1** [comment]
@mr-bagglesworth Thank you, once this has been merged I think I'll replace my button with your button of superior quality!!!!

**Comment by martinbagshaw** [comment]
Can people please avoid merging master into unmerged branches?
I just had to force push my branch to add my changes.
We should only be merging branches into master, not the other way around.

To get the updated master:
1 - commit branch you are working on
2 - checkout to master
3 - git pull origin master
4 - checkout to branch you were last working on
5 - git merge master + resolve any merge conflicts
6 - git push origin <your branch>

**Comment by wright1** [comment]
@mr-bagglesworth If we don't merge master into our branches how do we  make use of the new items added to master. Also doesn't it mean we're less likely to override master with something in our branch.

**Comment by wright1** [comment]
@mr-bagglesworth Thank you. Although what you just described I thought was merging master into your branch. My bad!!!!!!!!!!

**Review by MohammedYehia — APPROVED** [review]

**Pull request merged**
